pkgs/development/tools/misc/texinfo/6.0.nix

```diff
@@ -9,8 +9,11 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [ perl xz ]
+    ++ stdenv.lib.optional stdenv.isSunOS libiconv
     ++ stdenv.lib.optional interactive ncurses;
 
+  configureFlags = "AWK=${gawk}/bin/awk";
```

Comment:
Well, our `stdenv` is assumed to always provide (GNU) awk. `pkgs/stdenv/common-path.nix` contains a list of such packages. I believe the best approach is for all platforms to put GNU awk into stdenv, so we have better consistency.